src/Discovery/src/Eureka/EurekaDiscoveryClientExtension.cs

```diff
@@ -51,100 +51,103 @@ public bool IsConfigured(IConfiguration configuration, IServiceInfo serviceInfo
 
     internal void ConfigureEurekaServices(IServiceCollection services)
     {
-        services.AddOptions<EurekaClientOptions>().Configure<IConfiguration>((options, config) =>
+        services.AddOptions<EurekaClientOptions>().Configure<IConfiguration>((options, configuration) =>
         {
-            config.GetSection(EurekaClientOptions.EurekaClientConfigurationPrefix).Bind(options);
+            configuration.GetSection(EurekaClientOptions.EurekaClientConfigurationPrefix).Bind(options);
 
             // Eureka is enabled by default. If eureka:client:enabled was not set then check spring:cloud:discovery:enabled
-            if (options.Enabled && config.GetValue<bool?>($"{EurekaClientOptions.EurekaClientConfigurationPrefix}:enabled") is null &&
-                config.GetValue<bool?>(SpringDiscoveryEnabled) == false)
+            if (options.Enabled && configuration.GetValue<bool?>($"{EurekaClientOptions.EurekaClientConfigurationPrefix}:enabled") is null &&
+                configuration.GetValue<bool?>(SpringDiscoveryEnabled) == false)
             {
                 options.Enabled = false;
             }
-        }).PostConfigure<IConfiguration>((options, config) =>
+        }).PostConfigure<IConfiguration>((options, configuration) =>
         {
-            EurekaServiceInfo info = GetServiceInfo(config);
+            EurekaServiceInfo info = GetServiceInfo(configuration);
             EurekaPostConfigurer.UpdateConfiguration(info, options);
         });
 
         services.AddOptions<EurekaInstanceOptions>()
-            .Configure<IConfiguration>((options, config) => config.GetSection(EurekaInstanceOptions.EurekaInstanceConfigurationPrefix).Bind(options))
-            .PostConfigure<IServiceProvider>((options, serviceProvider) =>
-            {
-                var config = serviceProvider.GetRequiredService<IConfiguration>();
-                var appInfo = serviceProvider.GetRequiredService<IApplicationInstanceInfo>();
-                var inetOptions = config.GetSection(InetOptions.Prefix).Get<InetOptions>();
-                options.NetUtils = new InetUtils(inetOptions);
-                options.ApplyNetUtils();
-                const string endpointAssembly = "Steeltoe.Management.Endpoint";
-
-                if (ReflectionHelpers.IsAssemblyLoaded(endpointAssembly))
+            .Configure<IConfiguration>((options, configuration) =>
+                configuration.GetSection(EurekaInstanceOptions.EurekaInstanceConfigurationPrefix).Bind(options)).PostConfigure<IServiceProvider>(
```

Comment:
Was the splitting of `.PostConfigure` from it's param intentional? The diff is a bit hard to read, seems like there must have been a whitespace change that went with it

Reply:
I'm not sure I understand the question. This is what my diff tool shows when discarding leading whitespace:

![image](https://user-images.githubusercontent.com/104792814/186605933-0762a964-5390-465f-bfd4-784f03df509a.png)

I did the renames, then ran the auto-reformat, which changed indents in this case, but not the meaning of the code as far as I'm aware. By extracting nested logic in a separate method and using formatter directives, this can be cleaned up to the following:
```c#
// @formatter:wrap_chained_method_calls chop_always
// @formatter:keep_existing_linebreaks true

services
    .AddOptions<EurekaInstanceOptions>()
    .Configure<IConfiguration>((options, configuration) => configuration
        .GetSection(EurekaInstanceOptions.EurekaInstanceConfigurationPrefix)
        .Bind(options))
    .PostConfigure<IServiceProvider>((options, serviceProvider) => NewMethod(serviceProvider, options));

// @formatter:keep_existing_linebreaks restore
// @formatter:wrap_chained_method_calls restore
```

I can make that change, but what should "NewMethod" be named?

Reply:
This is what I saw initially
![image](https://user-images.githubusercontent.com/3947063/187290554-34fabb91-0802-47e9-9a35-6be91c701631.png)

After telling github to hide whitespace changes it's easier to read. The line break changes don't make it easier for _me_ to read to read the code, but it's not that big of a deal if that's what auto-format is doing